pkgs/by-name/tl/tlock/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "tlock";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "eklairs";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-O6erxzanSO5BjMnSSmn89w9SA+xyHhp0SSDkCk5hp8Y=";
+    fetchSubmodules = true;
+  };
+
+  vendorHash = "sha256-G402CigSvloF/SI9Wbcts/So1impMUH5kroxDD/KKew=";
+
+  doCheck = false;
+
+  excludedPackages = [
+    "bubbletea"
+    "scripts"
+  ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/eklairs/tlock/tlock-internal/constants.VERSION=v${version}"
+  ];
+
+  meta = with lib; {
+    mainProgram = pname;
+    license = licenses.mit;
+    homepage = "https://github.com/eklairs/tlock";
+    description = "Two-Factor Authentication Tokens Manager in Terminal";
+    changelog = "https://github.com/eklairs/tlock/releases/tag/v${version}";
+    maintainers = with maintainers; [ eklairs ];
```

```suggestion
    maintainers = with maintainers; [ eklairs ];
    platforms = lib.platforms.PLATFORM
```
Where `PLATFORM` is `linux`, `darwin`, `unix`, or something more exotic. If it's just x86_64 Linux, for instance, use `platforms = [ "x86_64-linux" ];`.